lib/activerecord-import/import.rb

```diff
@@ -591,8 +590,8 @@ def values_sql_for_columns_and_attributes(columns, array_of_attributes) # :nodoc
               connection_memo.quote(type_caster.type_cast_for_database(column.name, val))
             elsif column.respond_to?(:type_cast_from_user)                                   # Rails 4.2 and higher
               connection_memo.quote(column.type_cast_from_user(val), column)
-            else                                                                             # Rails 3.2, 4.0 and 4.1
-              if serialized_attributes.include?(column.name)
+            else                                                                             # Rails 3.1, 3.2, 4.0 and 4.1
+              if serialized_attributes.include?(column.name) && (options[:serialized] != false )
```

Comment:
This is the important part, avoid serialization if the serialized option is false. However, this only works for rails up to 4.1. If there's a way to make it work for all rails version it would be great.